### PR TITLE
wxOCaml.1.0 - via opam-publish

### PR DESCRIPTION
**Pull request by lefessan**
OCaml bindings to the wxWidgets graphical library

OCaml bindings to the wxWidgets graphical library

---
* Homepage: http://www.typerex.org/ocplib-wxOCaml.html
* Source repo: https://github.com/OCamlPro/ocplib-wxOCaml
* Bug tracker: https://github.com/OCamlPro/ocplib-wxOCaml/issues

---

Pull-request generated by opam-publish v0.3.1

### CODE REVIEW

**Comment by samoht** [comment]
Seems that removal is broken:

```
Processing  4/4: [wxOCaml: ocamlfind remove]
#=== ERROR while installing wxOCaml.1.0 =======================================#
# opam-version 1.2.2
# os           linux
# command      make
# path         /home/travis/.opam/system/build/wxOCaml.1.0
# compiler     system (4.01.0)
# exit-code    2
# env-file     /home/travis/.opam/system/build/wxOCaml.1.0/wxOCaml-7618-b1a82b.env
# stdout-file  /home/travis/.opam/system/build/wxOCaml.1.0/wxOCaml-7618-b1a82b.out
# stderr-file  /home/travis/.opam/system/build/wxOCaml.1.0/wxOCaml-7618-b1a82b.err
### stderr ###
# Makefile:12: config/Makefile: No such file or directory
# make: *** No rule to make target `config/Makefile'.  Stop.
```

**Comment by lefessan** [comment]
Hi Thomas,

Removal is based on "ocamlfind remove wxOCaml", so I don't think it is broken.
If Travis tries to build the package, it will probably fail during "make", since I don't think it has wxWidgets installed ? Or is opam-depext automatically called ?

File: packages/wxOCaml/wxOCaml.1.0.1/descr [hunk 1]
```diff
@@ -0,0 +1,4 @@
+OCaml bindings to the wxWidgets graphical library
+
+OCaml bindings to the wxWidgets graphical library
```
Comment (samoht) [inline packages/wxOCaml/wxOCaml.1.0.1/descr]: You don't need to duplicate the description in that file.

**Comment by samoht** [comment]
Still the same build error:

```
#=== ERROR while installing wxOCaml.1.0.1 =====================================#
# opam-version 1.2.2
# os           linux
# command      make
# path         /home/travis/.opam/system/build/wxOCaml.1.0.1
# compiler     system (4.00.1)
# exit-code    2
# env-file     /home/travis/.opam/system/build/wxOCaml.1.0.1/wxOCaml-7629-b1a82b.env
# stdout-file  /home/travis/.opam/system/build/wxOCaml.1.0.1/wxOCaml-7629-b1a82b.out
# stderr-file  /home/travis/.opam/system/build/wxOCaml.1.0.1/wxOCaml-7629-b1a82b.err
### stderr ###
# Makefile:12: config/Makefile: No such file or directory
# make: *** No rule to make target `config/Makefile'.  Stop.
```

**Comment by lefessan** [comment]
This error comes from the absence of wxWidgets on Travis (I just replayed it on my own computer). The ./configure script fails, and does not generate config/Makefile. Isn't it the problem with all C bindings where the C library is missing on Travis ?

**Comment by samoht** [comment]
Ha ok! Usually that kind of error is a bit less cryptic :-)

Can you clean-up the `descr` file and then I'm fine to merge your PR? Thanks!

**Comment by lefessan** [comment]
Is it ok now ?

**Comment by samoht** [comment]
Thanks!

**Pull request merged**